src/wh_13_1/ReadPlayers.java

```diff
@@ -0,0 +1,41 @@
+package wh_13_1;
+
+import java.util.*;
+
+public final class ReadPlayers {
+    private static List<Player> players = new ArrayList<>();
+
+    private ReadPlayers() {
+    }
+
+    static List<Player> readFromUser() {
+        Scanner sc = new Scanner(System.in);
+        String readLine;
+        do {
+            System.out.println("Podaj wynik kolejnego gracza <Imie Nazwisko wynik> (lub stop):");
+            readLine = sc.nextLine();
+            if ("stop".equals(readLine) && players.size() == 0) {
+                break;
+            }
+            if ("stop".equals(readLine)) {
+                continue;
+            } else {
+                players = insertData(readLine);
+            }
+        } while (!"stop".equals(readLine));
```

Comment:
troche duzo w tej petli sprawdzania czy "stop" zostalo podane. troche wporwadza to zamieszanie. Wydaje mi sie ze ladnie bedzie napisac tak:
```
while(true){
             readLine = sc.nextLine(); 
             if ("stop".equalsIgnoreCase(readLine)) { 
                    break;
             }
 ....
}
```

"stop" to tez dorby kandydat na stałą

Reply:
poprawione